doc/src/content/xdocs/spec.xml

```diff
@@ -328,15 +328,20 @@
     <section>
       <title>Data Serialization</title>
 
-      <p>Avro data is always serialized with its schema.  Files that
-	store Avro data should always also include the schema for that
-	data in the same file.  Avro-based remote procedure call (RPC)
-	systems must also guarantee that remote recipients of data
-	have a copy of the schema used to write that data.</p>
-
-      <p>Because the schema used to write data is always available
-	when the data is read, Avro data itself is not tagged with
-	type information.  The schema is required to parse data.</p>
+      <p>Avro data does not include type information.
+      The binary encoding does not include field names, either.  The resulting
+      tradeoff is that the serialized data is small, but the original schema of
+      the writer is always required in order to read Avro data correctly.</p>
+
+      <p>Therefore, files that store Avro data should always include the
+      schema for that data in the same file.  Avro-based remote procedure
+      call (RPC) systems must also guarantee that remote recipients of data
+      have a copy of the schema used to write that data.  In general, any
+      reader of Avro data must a schema that is structurally identical (as
```

Comment:
Sorry... I'm just trying to make the documentation clearer and easier to understand for people who are new to Avro (and in general). I suppose that talking about the possibility of a reader deserializing data into a different schema is not necessary within the section about serialization, so I can remove it.

However, I am confused about what you are saying. The section about the parsing canonical form  is the only section in the spec that I'm aware of where you can actually learn what it means for two schemas (the serializer's writing schema and the deserializer's reading schema) to be the "same" and therefore for it to be possible to use the one schema to read Avro written with another schema. That's what the whole section is about. I feel like I am missing something here... can you explain, so I can do a better job with this? How would it be possible to have structurally identical schemas (a term which you suggested but is not mentioned or defined in the spec, so how would you define it, if not via the parsing canonical form?) with different parsing canonical form? And doesn't that call into question statements from the docs such as "The schema is required to parse data."? These are the things I am trying to fix or clarify. Thanks for your help!